app/assets/javascripts/Results/context_menu.js.erb

```diff
@@ -16,35 +16,39 @@ var annotation_context_menu = {
   setup: function(annot_path, result_id, assignment_id, file_dl_path) {
     var menu_items = {
       new_annotation: {
-        title: 'New Annotation',
+        title: '<%= I18n.t('marker.annotation.context_menu.new') %>',
```

Comment:
I initially tried using `t`, but I was getting  a server error upon visiting the page:
```
undefined method `t' for #<#<Class:0x000000029bd320>:0x00000007abb6c8>
```